cmd/nerdctl/image_convert.go

```diff
@@ -229,6 +279,41 @@ func getESGZConvertOpts(cmd *cobra.Command) ([]estargz.Option, error) {
 	return esgzOpts, nil
 }
 
+func getNydusConvertOpts(cmd *cobra.Command) (*nydusconvert.PackOption, error) {
+	builderPath, err := cmd.Flags().GetString("nydus-builder-path")
+	if err != nil {
+		return nil, err
+	}
+	workDir, err := cmd.Flags().GetString("nydus-work-dir")
+	if err != nil {
+		return nil, err
+	}
+	chunkDictPath, err := cmd.Flags().GetString("nydus-chunk-dict")
+	if err != nil {
+		return nil, err
+	}
+	prefetchPatterns, err := cmd.Flags().GetString("nydus-prefetch-patterns")
+	if err != nil {
+		return nil, err
+	}
+	compressor, err := cmd.Flags().GetString("nydus-compressor")
+	if err != nil {
+		return nil, err
+	}
+	fsVersion, err := cmd.Flags().GetString("nydus-fs-version")
```

Comment:
I am suggesting only using fs-version 6, it's the future of nydus image format.

Reply:
Then option `nydus-fs-version` can be saved, it looks like too many options are added.

Reply:
But the fs-version 6 requires linux kernel version >= 5.19. If use version 6 as default, should we give an error when the user's kernel doesn't meet the requirements?

Reply:
Not really. :-) fs-version 6 is compatible with in-kernel fs EROFS from linux kernel 5.19 and beyond. But it still can work with linux or macos fuse.
In other words:
fs-version 5: FUSE / virtiofs
fs-version 6: FUSE / erofs+fscache / virtiofs

Reply:
Thanks! The `nydus-fs-version` can be removed and use `6` as the value of `FsVersion` of `PackOption`.

Reply:
BTW, do you think the option `nydus-work-dir` is necessary? Should we always use `/tmp` as its value?

Reply:
`/tmp` might consume all memories of the host. If I must choose one as temp workdir, I prefer locating it at `/var/cache` or `/var/lib/containerd-nydus/tmp` which is the default root directory of nydus-snapshotter